test/command/suite/sharding/logical_select/filter/accessor_optimize.test

```diff
@@ -0,0 +1,40 @@
+#$GRN_EXPR_OPTIMIZE=yes
+#@on-error omit
+plugin_register sharding
+#@on-error default
+
+table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenBigram   --normalizer NormalizerAuto
+
+table_create Users TABLE_PAT_KEY ShortText
+
+table_create Logs_20170315 TABLE_NO_KEY
+column_create Logs_20170315 timestamp COLUMN_SCALAR Time
+column_create Logs_20170315 user COLUMN_SCALAR Users
+column_create Logs_20170315 price COLUMN_SCALAR UInt32
+
+table_create Logs_20170316 TABLE_NO_KEY
+column_create Logs_20170316 timestamp COLUMN_SCALAR Time
+column_create Logs_20170316 user COLUMN_SCALAR Users
+column_create Logs_20170316 price COLUMN_SCALAR UInt32
+
+load --table Logs_20170315
+[
+{"timestamp": "2017/03/15 00:00:00", "user": "Suzuki", "price": 100},
+{"timestamp": "2017/03/15 01:00:00", "user": "Yamada", "price": 300},
+{"timestamp": "2017/03/15 02:00:00", "user": "Sato", "price": 200}
+]
+
+load --table Logs_20170316
+[
+{"timestamp": "2017/03/16 10:00:00", "user":  "Yamada", "price": 500},
+{"timestamp": "2017/03/16 11:00:00", "user":  "Yamada", "price": 600},
+{"timestamp": "2017/03/16 12:00:00", "user":  "Yamamoto", "price": 1000}
+]
+
+column_create Users logs_20170315_index COLUMN_INDEX Logs_20170315 user
+column_create Users logs_20170316_index COLUMN_INDEX Logs_20170316 user
+column_create Terms users_key COLUMN_INDEX|WITH_POSITION Users _key
+
+logical_select Logs \
+  --shard_key timestamp \
+  --filter 'price>400 && user._key=="Yamamoto"'
```

Comment:
It seems that it is better to contain `mutistage` or something in xxx.test file name because `user._key` is important part, doesn't it.